.github/workflows/supply-chain-security-validation.yaml

```diff
@@ -6,6 +6,16 @@ concurrency:
 on:
   workflow_call:
     inputs:
+      checkout-submodules-mode:
+        description: 'Submodules mode (false, true, or recursive)'
+        default: "false"
+        type: string
+        required: false
+      checkout-submodules-fetch-depth:
+        description: 'Fetch depth (0 for full history)'
+        default: "1"
+        type: string
+        required: false
```

Comment:
This is used in `fetch-depth`, which is not for submodules but for the repository itself. So, if we were to expose this, we should name it correctly. Is there a need to expose this?